tests/headers/structs.h

```diff
@@ -29,7 +29,7 @@ struct recursive_struct {
 typedef struct tagWNDCLASSEXA {
     int         cbClsExtra;
     int         cbWndExtra;
-} WNDCLASSEXA, *PWNDCLASSEXA, near *NPWNDCLASSEXA, far *LPWNDCLASSEXA;
```

Comment:
Are far and near tested somewhere else ? or do you have a reason to remove support for them ?